src/data/markdown/docs/30 xk6-browser/01 xk6-browser/09 Locator/textcontent--options--.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'innerText([options])'
```

Comment:
```suggestion
title: 'textContent([options])'
```

This messes up the left-side menu, which renders two `innerText()` links, but the `textContent()` page is unreachable. :facepalm: It should probably raise an error if a duplicate link is found...

Reply:
Resolved in: 2a3d1d6c25938185f76234a36c469b51185ef9ba